mobile/src/main/java/rs/readahead/washington/mobile/views/activity/viewer/VaultActionsHelper.kt

```diff
@@ -107,7 +107,10 @@ object VaultActionsHelper {
                     getString(R.string.action_cancel),
                     object : BottomSheetUtils.ActionConfirmed {
                         override fun accept(isConfirmed: Boolean) {
-                            exportMediaFile()
+                            if (isConfirmed)
+                                exportMediaFile()
+                            else
+                                return
```

Comment:
No need for an 'else' block; just return when isConfirmed is false, an make sure to add code  Braces :

```
if (isConfirmed) {
            exportMediaFile()
        }

```